### PR TITLE
[RTM] Disable html5 validation if button with nosave css class

### PR DESCRIPTION
**Pull request by zonky2**
Disable html5 validation if button with nosave css class

related with MM 2.2 MR "Add css class nosave at buttons if checkbox activated"

### CODE REVIEW

File: src/Resources/contao/templates/dcfe_general_edit.html5 [hunk 1]
```diff
@@ -26,12 +27,12 @@
       enctype="<?= $this->enctype; ?>"
 >
     <div class="formbody_edit">
-        <input type="hidden" name="FORM_SUBMIT" value="<?= specialchars($this->table); ?>"/>
+        <input type="hidden" name="FORM_SUBMIT" value="<?= \StringUtil::specialchars($this->table); ?>"/>
```
Comment (dmolineus) [inline src/Resources/contao/templates/dcfe_general_edit.html5]: Please use FQCN here, `\Contao\StringUtil`

File: src/Resources/contao/templates/dcfe_general_edit.html5 [hunk 2]
```diff
@@ -26,12 +27,12 @@
...
         <input type="hidden" name="REQUEST_TOKEN" value="<?= REQUEST_TOKEN; ?>"/>
         <?php if ($this->error): ?><p class="tl_error"><?= implode('<br />', $this->error); ?></p><?php endif; ?>
         <?php foreach ($this->fieldsets as $arrFieldset): ?>
             <?php if ($arrFieldset['legend']): ?>
-                <fieldset id="pal_<?= specialchars($arrFieldset['legend']); ?>"
+                <fieldset id="pal_<?= \StringUtil::specialchars($arrFieldset['legend']); ?>"
```
Comment (dmolineus) [inline src/Resources/contao/templates/dcfe_general_edit.html5]: Please use FQCN here, `\Contao\StringUtil`

File: src/Resources/contao/templates/dcfe_general_edit.html5 [hunk 3]
```diff
@@ -49,3 +50,14 @@
         </div>
     </div>
 </form>
+<?php
```
Comment (dmolineus) [inline src/Resources/contao/templates/dcfe_general_edit.html5:53]: I don't like the solution. The class `notsave` is part of MetaModels and not used by the DC General anywhere. Therefore I think the JS schould be part of MM frontend editing also.

**Review by dmolineus — CHANGES_REQUESTED** [review]

File: src/Resources/contao/templates/dcfe_general_edit.html5 (hunk 2 above)
Comment (zonky2) [reply src/Resources/contao/templates/dcfe_general_edit.html5]: fixed

**Pull request merged**